### PR TITLE
LPS-106345 Update experience priority

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceSelector.js [hunk 1]
```diff
@@ -36,6 +36,42 @@ import {
 import ExperienceModal from './ExperienceModal';
 import ExperiencesList from './ExperiencesList';
 
+/**
+ * It produces an object with a target and subtarget keys indicating what experiences
+ * should change to change the priority of target priority.
+ */
+function experiencePrioritySwap(
```
Comment (dgarciasarai) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceSelector.js]: I don't know if this is a good naming 🤔

Comment (dgarciasarai) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceSelector.js]: `changeExperiencePriority`? We can discuss this

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js [hunk 2]
```diff
@@ -172,4 +180,156 @@ describe('ExperienceToolbarSection', () => {
...
+
+		/**
+		 * Top Experience cannot be prioritized
+		 */
+		expect(topExperiencePriorityButton.disabled).toBe(true);
+
+		/**
+		 * Bottom Experience can be prioritized
+		 */
+		expect(bottomExperiencePriorityButton.disabled).toBe(false);
```
Comment (dgarciasarai) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js]: bottom, button 😂 👍

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js (hunk 2 above)
Comment (andresfulla) [reply modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js]: The button of the Experience at the bottom 😋

**Comment by dgarciasarai** [comment]
Reviewed

**Pull request closed without merging**